Exercise_A/index.html

```diff
@@ -15,8 +15,20 @@
     />
   </head>
   <body>
-    <!-- CODE HERE -->
-
+    <div class="List Practice">
+      <ol>
+        <li>Monday</li>
+        <li>Tuesday</li>
+        <li>Wednesday</li>
+        <li>Thursday</li>
+        <li>Friday</li>
+        <li>Saturday</li>
+        <li>Sunday</li>
+      </ol>
+    </div>
+    <div class="Create Two Pages">
+      <a href="./About.html" target="_black"><img src="Ame.jfif" alt="pretty Ame"></a>
+    </div>
```

Comment:
Great work!  I would use a comment instead of a class to label where the exercise begins.